posthog/types.py

```diff
@@ -4,8 +4,28 @@
 from posthog.models.filters.path_filter import PathFilter
 from posthog.models.filters.retention_filter import RetentionFilter
 from posthog.models.filters.stickiness_filter import StickinessFilter
-from posthog.schema import FunnelsQuery, LifecycleQuery, PathsQuery, RetentionQuery, StickinessQuery, TrendsQuery
+from posthog.schema import (
+    FunnelsQuery,
+    LifecycleQuery,
+    PathsQuery,
+    RetentionQuery,
+    StickinessQuery,
+    TrendsQuery,
+    WebTopSourcesQuery,
+    WebTopClicksQuery,
+    WebTopPagesQuery,
+)
 
 FilterType = Union[Filter, PathFilter, RetentionFilter, StickinessFilter]
 
-InsightQueryNode = Union[TrendsQuery, FunnelsQuery, RetentionQuery, PathsQuery, StickinessQuery, LifecycleQuery]
+Node = Union[
+    TrendsQuery,
+    FunnelsQuery,
+    RetentionQuery,
+    PathsQuery,
+    StickinessQuery,
+    LifecycleQuery,
+    WebTopSourcesQuery,
+    WebTopClicksQuery,
+    WebTopPagesQuery,
+]
```

Comment:
Hmm... I was thinking this could just be `posthog.schema.Node`, the root of all `*Query` classes. Otherwise I'd name this here something different to differentiate from that root class.

Reply:
posthog.schema.Node doesn't currently exist in python land. I got the schema.ts -> json -> py script to export it, but it isn't compatible (channeling Liskov, I can see why not, but I'm also not sure why what we have at the moment works either). It's probably easiest to just change the name :)